src/navigation/AppNavigator.tsx

```diff
@@ -21,11 +21,15 @@ import {NotificationSettingsScreen} from 'screen/NotificationSettingsScreen';
 import {Icon, TopNavigationAction} from '@ui-kitten/components';
 import {DashboardStackParamList, DrawerParamList} from 'src/navigation/navigation';
 import globalStyles from 'src/styles';
-import {submitTipScreen} from 'src/navigation/routeKeys';
+import {useFirebase} from 'src/hook/useFirebase';
 
 const DashboardStack = createStackNavigator<DashboardStackParamList>();
 
 function DashboardStackNavigator() {
+  // needs to be called inside a navigator in order to
+  // have access to navigator context
+  useFirebase();
```

Comment:
It would be safer to make useFirebase to throw an error if the context is undefined instead of the comments

Reply:
done ✅